integration/pkg/api/osrm/table/response.go

```diff
@@ -0,0 +1,13 @@
+package table
+
+import "github.com/Telenav/osrm-backend/integration/pkg/api/osrm/route"
+
+// Response represents OSRM api v1 table response.
+type Response struct {
+	Code         string            `json:"code"`
+	Message      string            `json:"message,omitempty"`
+	Sources      []*route.Waypoint `json:"sources"`
```

Comment:
`table` API uses `route.Waypoint` still looks strange.

Reply:
Move Waypoint to package `osrmtype`.  I think common definitions like `coordinate`, `segment` could be moved here.